proto/kvrpcpb.proto

```diff
@@ -459,7 +459,7 @@ message RawBatchPutRequest {
     Context context = 1;
     repeated KvPair pairs = 2;
     string cf = 3;
-    uint64 ttl = 4;
+    repeated uint64 ttl = 4;
```

Comment:
```suggestion
    repeated uint64 ttls = 4;
```

Reply:
it is not compatile with old version if it is modified to ttls, as you said before.

Reply:
renaming in protobuf is compatible as long as the number remains the same.

I don't know that changing singular field to repeated will be compatible, and thanks @disksing for pointing out.

Reply:
> renaming in protobuf is compatible as long as the number remains the same.

It is wire format compatible, but the generated code will change API names, and cause compile fail when upgrading.

Reply:
> cause compile fail when upgrading 

This is intended. The new tikv server and client should re-implement the logic on TTL.